unlock-app/src/registerServiceWorker.js

```diff
@@ -40,12 +40,11 @@ export default function register() {
 
         // Add some additional logging to localhost, pointing developers to the
         // service worker/PWA documentation.
-        navigator.serviceWorker.ready.then(() => {
-          console.log(
-            'This web app is being served cache-first by a service ' +
-              'worker. To learn more, visit https://goo.gl/SC7cgQ'
-          )
-        })
+        await navigator.serviceWorker.ready
```

Comment:
can you wrap in a try/catch that throws away the error to match the promise behavior?

Reply:
The existing behavior didn't have a catch here, i'd like to ensure that the replacements here are as they were to reduce the likelihood of unintended side-effects